api/static/css/docs.css

```diff
@@ -0,0 +1,13 @@
+.hidden-label {
```

Comment:
`.hidden-label` is in arxiv-base, shouldn't have to declare it here (and the inline page styles belong here)